### ISSUE TITLE
Responsive Issue with Buttons in ngx-extended-pdf-viewer on Small Screens

### ISSUE THREAD

**Issue by AmmarBaig135** [stephanrauh/ngx-extended-pdf-viewer#2651]
### Description: 
I'm experiencing a responsiveness issue with the ngx-extended-pdf-viewer component in my Angular project. I have enabled only the showDownloadButton, showPrintButton, and showDrawEditor options. When the screen width falls below 590px, all three buttons disappear from the view.

To resolve this, I applied the following CSS media query, which successfully makes the print and download buttons visible but does not work for the draw editor button.

```html
<ngx-extended-pdf-viewer *ngIf="selectedFile" [src]="selectedFile" [height]="'600px'" 
    [filenameForDownload]="consentDocumentName" [textLayer]="true" [filenameForDownload]="fileName" 
    [showToolbar]="true" [showSidebarButton]="false" [showFindButton]="false" 
    [showPagingButtons]="false" [showDrawEditor]="!viewFile" [showStampEditor]="false" 
    [showTextEditor]="false" [showZoomButtons]="false" [showPresentationModeButton]="false" 
    [showOpenFileButton]="false" [showPrintButton]="true" [showDownloadButton]="true" 
    [showSecondaryToolbarButton]="false" [showRotateButton]="false" [showHandToolButton]="false" 
    [showScrollingButton]="false" [showSpreadButton]="false" [showPropertiesButton]="false" 
    [showHighlightEditor]="false">
</ngx-extended-pdf-viewer>
```
```css
@media (max-width: 590px){
::ng-deep button#primaryEditorInk,
  ::ng-deep button#primaryEditorInk * {
    display: block !important;
  }


  ::ng-deep button#printButton,
  ::ng-deep button#printButton * {
    display: block !important;
  }

  ::ng-deep button#downloadButton,
  ::ng-deep button#downloadButton * {
    display: block !important;
  }
  }
```
### Observed Behavior:
- The print and download buttons reappear as expected for screen sizes below 590px.
- However, the draw editor button (with button#primaryEditorInk selector) does not appear, despite using the same display: block !important; styling.

### Expected Behavior:
All three buttons (download, print, and draw editor) should be visible on screen widths below 590px.

**Comment by victorDECLAU** [comment]
Hello,
Are you sure `[showDrawEditor]="!viewFile"` is computed as `[showDrawEditor]="true"` in you case ? 
If not, the button isn't rendered, and therefore can't be styled.

**Comment by stephanrauh** [comment]
You're right, both `[showPrintButton]` and `[showDrawEditor]` were buggy. The bugfix has landed with version 22.0.0-alpha.10.

However, there's also a misunderstanding. Setting `[showXXXButton]="true"` doesn't mean the button is always visible. It only means the default classes are used. That's why the download button vanishes. That's the intended behavior.

Setting 
```html
[showDownloadButton]="'always-visible'"
[showDrawEditor]="'always-visible'"
[showPrintButton]="'always-visible'"
```
solves the issue (after updating to version 22.0.0-alpha.10).
